build.gradle

```diff
@@ -1,13 +1,12 @@
 plugins {
-	id 'fabric-loom' version '1.8-SNAPSHOT'
-	id 'maven-publish'
+	id "fabric-loom" version "1.8-SNAPSHOT"
+	id "maven-publish"
```

Comment:
I don't mean to start a war or anything, but I think single quotes are better than double quotes where possible in Groovy. Double quotes create a `GString` (which is coerced to a `String` almost everywhere), whereas single quotes directly create a `String`. IMO you should only use double quotes when formatting (with `$`) is used.